code/datums/mutations.dm

```diff
@@ -147,6 +168,10 @@ var/thanks_tobba = 'icons/fonts/runescape_uf.ttf'
 	if(..())
 		return
 	owner.status_flags |= CANSTUN | CANWEAKEN | CANPARALYSE | CANPUSH
+	owner.client.prefs.features["mcolor"] = naturalcolor
```

Comment:
oh my fucking god blobber please
even if you set dna.features["mcolor"] the problem is you're still overlaying "visual_indicators" so mcolor doesn't look like it's working, though it probably is.
axe off the "visual indicators" bit entirely since it's still incompatible with dismemberment anyway.